src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -183,6 +183,15 @@ internal void SetRefreshCredentialNeeded()
             _refreshCredentialNeeded = true;
         }
 
+        ~SecureChannel()
+        {
+            if(_sslAuthenticationOptions.AlpnProtocolsHandle.IsAllocated)
+            {
+                _sslAuthenticationOptions.AlpnProtocolsHandle.Free();
+            }
+
```

Comment:
Nit: unnecessary blank line (in addition to the missing space after the if above

Reply:
Yeah sorry was late, thought you meant a new line ... I have fixed.